src/ralph/accounts/views.py

```diff
@@ -58,30 +59,36 @@ def get_context_data(self, **kwargs):
 class InventoryTagView(View):
     http_method_names = ['post']
 
+    @staticmethod
+    def _add_tags(request, asset, tags):
+        asset.tags.add(*tags)
+        with reversion.create_revision():
+            asset.save()
+            reversion.set_user(request.user)
+            reversion.set_comment('Added tags {}'.format(', '.join(tags)))
```

Comment:
maybe ``Added inventory tags {}``? Then we have easy way to distinguish between normal adding and adding by self-inventory mechanism.